src/browser/web-view.js

```diff
@@ -341,6 +342,8 @@ const viewFrame = (model, address) =>
     onMozBrowserScrollAreaChanged: on(address, decodeScrollAreaChange),
   });
 
+const asForInput = asFor('input');
```

Comment:
@TODO wrapping action in `For` here means that by the time the action is sent, it is triple-wrapped. Is that what we want?